src/python/deepgnn/graph_engine/snark/distributed.py

```diff
@@ -16,16 +16,18 @@
 class Client(ge_snark.Client):
     """Distributed client."""
 
-    def __init__(self, servers: List[str], ssl_cert: str = None):
+    def __init__(
+        self, servers: List[str], ssl_cert: str = None, delayed_start: bool = False
+    ):
         """Init snark client to wrapper around ctypes API of distributed graph."""
         self.logger = get_logger()
-        self.logger.info(f"servers: {servers}. SSL: {ssl_cert}")
-        self.graph = client.DistributedGraph(servers, ssl_cert)
+        # self.logger.info(f"servers: {servers}. SSL: {ssl_cert}")
```

Comment:
why logging is removed?

Reply:
I added silent mode, it is because graph client gets re-initialized so many times the logs are too much.